src/main/java/com/farao_community/farao/dichotomy/Index.java

```diff
@@ -90,8 +87,9 @@ boolean precisionReached() {
         if (higherSecureStep != null && Math.abs(higherSecureStep.stepValue() - maxValue) < EPSILON) {
```

Comment:
Can't you just test higherSecureStep.stepValue() >= maxValue ? It feels a bit too specifc to test equality

Reply:
The thing is it cannot be higher than maxValue, so it is a bit strange to let it appear as a possibility no ?

Reply:
And with doubles, checking equality like that may create issues ...

Reply:
Yes this question was a bit related with our discussion on the two different questions : did we reach the limits and did we reach the precision. But that's ok for me like that !